src/core_modules/capture-ui/IconButton/IconButton.component.js

```diff
@@ -0,0 +1,33 @@
+// @flow
```

Comment:
We will need replace the old material-UI IconButton that is being used with this one (or at least this is a start, might need to expand on this to fulfill the requirements). Let's do that as part of the already existing ticket we have.